### PR TITLE
Add note to DPDK README that PSA egress is not implemented

### PR DESCRIPTION
**Pull request by jafingerhut**

### CODE REVIEW

**Comment by jafingerhut** [comment]
@usha1830 For your consideration on whether to approve.  I can change it to be more accurate, if I have made a mistake.

**Review by usha1830 — APPROVED** [review]
This looks fine to me

File: backends/dpdk/README.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ To load the 'spec' file in dpdk follow the instructions in the
 - Parser Value Sets
 
 ### Unsupported PSA externs and features
+- egress parser, control, and deparser are not implemented, only
+  ingress parser, control, and deparser.  There is no traffic manager.
```
Comment (rst0git) [inline backends/dpdk/README.md:39]: If I understand correctly, "traffic manager" is described in the PSA specification as "Packet buffer and Replication Engine (PRE)". Is that correct?

Comment (jafingerhut) [reply backends/dpdk/README.md]: I have added another commit to this PR that mentions packet buffer and packet replication engine, to make the terms easier to find in the PSA spec.

**Review by rst0git — APPROVED** [review]
LGTM

**Pull request merged**